src/javascripts/components/views/viewHelper.js

```diff
@@ -0,0 +1,40 @@
+import auth from '../auth/auth';
+import menuView from './menuView';
+import reservationView from './reservationView';
+import staffView from './staffView';
+
+const viewHelper = (id) => {
+  switch (id) {
+    // targeting nav link id, returning respective link view
+    case 'menuLink':
+      return menuView.menuView();
+    case 'reservationLink':
+      return reservationView.reservationView();
+    case 'staffLink':
+      return staffView.staffView();
+    case 'home':
+      return console.warn('homeView');
+    default:
+      return console.warn('nothing clicked');
+  }
+};
+
+const viewListener = (view, user) => {
+  viewHelper(view);
+
+  // targeting nav link id on click event to print nav link respective view
+  $('body').on('click', 'a.nav-link', (e) => {
+    e.stopImmediatePropagation();
+    viewHelper(e.currentTarget.id);
+  });
+
+  $('body').on('click', '#userLink', () => {
+    if (user) {
```

Comment:
Does this only work on click of this link? What happens with the login button if a authed user clicks another view?

Reply:
Just fixed these couple of issues, how should I go about pushing these up?

Reply:
Ok @drteresavasquez, the thought behind that conditional was that a click on the same button would serve two purposes: if the user was logged in, they would see the logout screen, and if the user was not logged in, they would instead see a login screen. We changed it so that the conditional is in authData.js and now we are adding/removing classes to change the functionality of the user button.

Reply:
@chrislojacono update this PR with the changes and deploy and I’ll approve once tested and code reviewed.